packages/file_selector/file_selector_macos/lib/file_selector_macos.dart

```diff
@@ -88,6 +88,22 @@ class FileSelectorMacOS extends FileSelectorPlatform {
     );
   }
 
+  @override
+  Future<List<String?>> getDirectoriesPaths({
```

Comment:
Can you review this? @eugerossetto made a change to the method in platform_interface.
I think this should be the actual implementation:
```dart
Future<List<String>?> getDirectoriesPaths({
    String? initialDirectory,
    String? confirmButtonText,
  }) async {
      return _channel.invokeListMethod<String>(
      'getDirectoriesPaths',
      <String, dynamic>{
        'initialDirectory': initialDirectory,
        'confirmButtonText': confirmButtonText,
        'multiple': true,
      },
    );
  }
```